frontend/src/api/board.js

```
@@ -0,0 +1,226 @@
+    const response = await api.get('/api/board', { 
+      params: { ...params, bookmarked: true } 
+    })
+    return response.data
+  } catch (error) {
+    throw error
+  }
+}
+
+// 게시글 상세 조회
+export const getBoardDetail = async (boardId) => {
+  try {
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)